database_setup.py

```diff
@@ -0,0 +1,36 @@
+from psycopg2 import connect
+import sys
+import os
+import subprocess
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+
+os.system('sudo -u postgres psql -c \"CREATE ROLE aditya WITH LOGIN PASSWORD \'password\';\"')
+
+con = None
+con = connect(database='postgres',user='aditya', host = 'localhost', password='password')
+
+database = "students"
+con.set_isolation_level(ISOLATION_LEVEL_AUTOCOMMIT)
+cur = con.cursor()
+
+output = subprocess.check_output('sudo -u postgres -i psql -l | grep students | wc -l ',shell=True)
```

Comment:
This should be done with query instead of subprocess, cause of same reason as above.